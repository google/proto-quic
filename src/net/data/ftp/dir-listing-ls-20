drwxrwxr-x   17 ftp      ftp          4096 Nov 01 16:27 .
drwxr-xr-x    7 ftp      ftp          4096 Apr 03  2010 ..
drwxrwxrwx    5 ftp      ftp          4096 May 26 15:51 2012_-_2012-(2009)-[1080p]_[BD]
-rw-rw-rw-    1 ftp      ftp          4931 Jun 08 15:24 _READ_ME.txt
drwxrwxrwx    5 ftp      ftp          4096 Nov 01 16:27 Без_лица_-_Face_Off-(1997)-[1080p]_[BD]
drwxrwxrwx    4 ftp      ftp          4096 Jan 22  2010 Вверх_-_Up-(2009)-[1080p]_[BD]
drwxrwxrwx    5 ftp      ftp          4096 May 27 18:12 Вспомнить_все_-_Total_Recall-(1990)-[1080p]_[BD]
drwxrwxrwx    3 ftp      ftp          4096 May 21 14:28 Законопослушный_гражданин_[Расширенная_версия]_-_Law_Abiding_Citizen_[Unrated_Edition]-(2009)-[1080p]_[BD_Remux]
drwxrwxrwx    5 ftp      ftp          4096 Jan 21  2010 Монстро_-_Cloverfield-(2008)-[1080p]_[BD]
drwxrwxrwx    4 ftp      ftp          4096 May 26 13:43 Пандорум_-_Pandorum-(2009)-[1080p]_[BD]
drwxrwxrwx    4 ftp      ftp          4096 May 27 14:18 Последний_самурай_-_The_Last_Samurai-(2003)-[1080p]_[BD]
drwxrwxrwx    6 ftp      ftp          4096 Jan 27  2010 Район_9_-_District_9-(2009)-[1080p]_[BD]
drwxrwxrwx    7 ftp      ftp          4096 Jan 01  2010 Рокки_Антология_-_Rocky_The_Undisputed_Collection-(1976_1979_1982_1985_1990)-[1080p]_[BD]
drwxrwxrwx    4 ftp      ftp          4096 May 31 12:57 Суррогаты_-_Surrogates-(2009)-[1080p]_[BD]
drwxrwxrwx    4 ftp      ftp          4096 Jan 28  2010 Тройной_Форсаж-Токийский_Дрифт_-_The_Fast_and_the_Furious-Tokyo_Drift-(2006)-[1080p]_[BD]
drwxrwxrwx    4 ftp      ftp          4096 Jan 08  2010 Форсаж_-_The_Fast_and_the_Furious-(2001)-[1080p]_[BD]
drwxrwxrwx    4 ftp      ftp          4096 Jan 06  2010 Форсаж_2_-_2_Fast_2_Furious-(2003)-[1080p]_[BD]
drwxrwxrwx    4 ftp      ftp          4096 Jun 08 15:26 Форсаж_4_-_Fast_&_Furious-(2009)-[1080p]_[BD]
