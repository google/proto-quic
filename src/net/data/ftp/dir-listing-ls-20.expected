d
.
-1
1994
11
1
16
27

d
..
-1
2010
4
3
0
0

d
2012_-_2012-(2009)-[1080p]_[BD]
-1
1994
5
26
15
51

-
_READ_ME.txt
4931
1994
6
8
15
24

d
Без_лица_-_Face_Off-(1997)-[1080p]_[BD]
-1
1994
11
1
16
27

d
Вверх_-_Up-(2009)-[1080p]_[BD]
-1
2010
1
22
0
0

d
Вспомнить_все_-_Total_Recall-(1990)-[1080p]_[BD]
-1
1994
5
27
18
12

d
Законопослушный_гражданин_[Расширенная_версия]_-_Law_Abiding_Citizen_[Unrated_Edition]-(2009)-[1080p]_[BD_Remux]
-1
1994
5
21
14
28

d
Монстро_-_Cloverfield-(2008)-[1080p]_[BD]
-1
2010
1
21
0
0

d
Пандорум_-_Pandorum-(2009)-[1080p]_[BD]
-1
1994
5
26
13
43

d
Последний_самурай_-_The_Last_Samurai-(2003)-[1080p]_[BD]
-1
1994
5
27
14
18

d
Район_9_-_District_9-(2009)-[1080p]_[BD]
-1
2010
1
27
0
0

d
Рокки_Антология_-_Rocky_The_Undisputed_Collection-(1976_1979_1982_1985_1990)-[1080p]_[BD]
-1
2010
1
1
0
0

d
Суррогаты_-_Surrogates-(2009)-[1080p]_[BD]
-1
1994
5
31
12
57

d
Тройной_Форсаж-Токийский_Дрифт_-_The_Fast_and_the_Furious-Tokyo_Drift-(2006)-[1080p]_[BD]
-1
2010
1
28
0
0

d
Форсаж_-_The_Fast_and_the_Furious-(2001)-[1080p]_[BD]
-1
2010
1
8
0
0

d
Форсаж_2_-_2_Fast_2_Furious-(2003)-[1080p]_[BD]
-1
2010
1
6
0
0

d
Форсаж_4_-_Fast_&_Furious-(2009)-[1080p]_[BD]
-1
1994
6
8
15
26
